README.md

```diff
@@ -44,12 +46,10 @@ The Details object specifies information about the event. Details are returned i
 }
 ```
 
-<br>
-
 - ### Venue
 A Venue object contains information about various venues where Events are held.
 
-```
+```json
```

Comment:
I've found that using 'js' rather than 'json' makes the comments look better! Although it changes the key colour as well. Up to you, here's an comparison:

``` json
{
  "using": "json" // comment is red
}
```

``` js
{
  "using": "js" // comment is regular
}
```